pkg/cmd/pulumi/org.go

```diff
@@ -0,0 +1,150 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/pulumi/pulumi/pkg/v3/backend/display"
+	"github.com/pulumi/pulumi/pkg/v3/backend/httpstate"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+)
+
+func newOrgCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "org",
+		Short: "Manage Organisation configuration",
+		Long: "Manage Organisation configuration.\n" +
+			"\n" +
+			"Use this command to manage any specific organization configuration, " +
+			"e.g. setting the default organization for a backend",
+		Args: cmdutil.NoArgs,
+	}
+
+	cmd.AddCommand(newOrgSetCmd())
+	cmd.AddCommand(newOrgGetCmd())
+
+	return cmd
+}
+
+func newOrgSetCmd() *cobra.Command {
+	var orgName string
+
+	var cmd = &cobra.Command{
+		Use:   "set [NAME]",
+		Args:  cmdutil.MaximumNArgs(1),
+		Short: "Set the default org for the current backend",
+		Long: "Set the default org for the current backend.\n" +
+			"\n" +
+			"This command is used to set the default organization in which to create \n" +
+			"projects and stacks for the current backend.\n" +
+			"\n" +
+			"Currently, only the Pulumi SaaS and Pulumi Self Hosted solutions, support setting \n" +
+			"organizations. If you try and set a default organization for a backend that does not \n" +
+			"support create organizations, then an error will be returned by the CLI",
+		Run: cmdutil.RunFunc(func(cmd *cobra.Command, args []string) error {
+			displayOpts := display.Options{
+				Color: cmdutil.GetGlobalColorization(),
+			}
+
+			if len(args) > 0 {
+				if orgName != "" {
+					return errors.New("only one of --org-name or argument may be specified, not both")
+				}
+				orgName = args[0]
+			}
+
+			currentBe, err := currentBackend(displayOpts)
+			if err != nil {
+				return err
+			}
+			if !currentBe.SupportsOrganizations() {
+				return fmt.Errorf("Unable to create a default organization for backend type: %s",
+					currentBe.Name())
+			}
+
+			if _, ok := currentBe.(httpstate.Backend); ok {
+				cloudURL, err := workspace.GetCurrentCloudURL()
+				if err != nil {
+					return err
+				}
+				if err := httpstate.SetDefaultOrg(cloudURL, orgName); err != nil {
+					return err
+				}
+			}
+
+			return nil
+		}),
+	}
+
+	cmd.PersistentFlags().StringVar(&orgName,
+		"org-name", "", "The name of the default organization")
+
+	return cmd
+}
+
+func newOrgGetCmd() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "get [NAME]",
+		Args:  cmdutil.MaximumNArgs(1),
+		Short: "Get the default org for the current backend",
+		Long: "Get the default org for the current backend.\n" +
+			"\n" +
+			"This command is used to get the default organization for which and stacks are created in " +
+			"the current backend.\n" +
```

Comment:
Currently this command returns both the default org and the name of the backend.  Should this return just the default org as a plain string (for scripting scenarios), and allow `pulumi org` to return the richer information this command currently returns?